README.md

```diff
@@ -988,6 +989,36 @@ targets:
     releaseRepoName: 'unity'
 ```
 
+### Maven central (`maven`)
+
+PGP signs and publishes packages to Maven Central.
+
+**Environment**
+
+| Name                     | Description                                                                                                                       |
+| ------------------------ | --------------------------------------------------------------------------------------------------------------------------------- |
+| `OSSRH_USERNAME`         | Username of Sonatype repository.                                                                                                  |
+| `OSSRH_PASSWORD`         | Password of Sonatype repository.                                                                                                  |
+| `MAVEN_CENTRAL_USERNAME` | Username of Maven Central.                                                                                                        |
+| `MAVEN_CENTRAL_PASSWORD` | Password of Maven Central.                                                                                                        |
+| `DISTRIBUTIONS_PATH`     | **optional** Path to the directory with unzipeed distribution packages. Defaults to `distributions/`.                             |
+| `SETTINGS_PATH`          | **optional** Path to the Maven `settings.xml`, containing Maven Central username and API key. Defaults to `scripts/settings.xml`. |
```

Comment:
These names are too generic. We should scope them to `MAVEN_` or `CRAFT_` (or both)